lib/bootloader_setup.pm

```diff
@@ -383,7 +383,8 @@ sub uefi_bootmenu_params {
     assert_screen("grub2-enter-edit-mode", 30) if is_jeos;
     # Kiwi in TW uses grub2-mkconfig instead of the custom kiwi config
     # Locate gfxpayload parameter and update it
-    if (is_jeos && (is_tumbleweed || is_sle('>=15-sp1') || is_leap('>=15.1'))) {
+    if (is_jeos && (!is_sle('=12-SP5') || is_opensuse)) {
+
```

Comment:
unrelated change

Reply:
No it isn't.
This is an improvement on https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18622#pullrequestreview-1870323749

Reply:
I was only relating to the blank line